README.md

```diff
@@ -34,6 +34,29 @@ make test
 See [`CONTRIBUTING.md`](CONTRIBUTING.md) for details on how to contribute to
 this project.
 
+The code in this project is formatted with `clang-format(1)`, and our CI builds
+will check that the code matches the format generated by this tool before
+accepting a pull request.
+Please configure your editor or IDE to use the Google style for indentation and
+whitespace.
+If you need to reformat one or more files you can simply:
+
+```console
+$ clang-format -i <file>....
```

Comment:
We don't use `$` in the command sequence above, might as well drop it here so that it's easier to copy-paste.